src/content/docs/security/security-privacy/compliance/fedramp-compliant-endpoints.mdx

```diff
@@ -12,27 +12,35 @@ redirects:
 
 This document provides information on FedRAMP-compliant endpoints in New Relic. For more information about our security accreditation for the Federal Risk and Authorization Management Program (FedRAMP), see our [data encryption](/docs/security/new-relic-security/compliance/data-encryption) documentation.
 
-<Callout variant="important">
- 
- **Customers' FedRAMP Obligations**
- 
-  Customers must meet all of the following requirements for New Relic’s FedRAMP environment:
-  
-  1. **New Relic-Approved Customers**: New Relic’s FedRAMP-Moderate authorized environment is only available to New Relic-approved customers. For more information, contact your New Relic account representative;
-       
-  2. **Order Form**: Customer’s Order Form with New Relic must include customer’s eligibility for FedRAMP;
-  
-  3. **Subscription Level**: Customer must have a current and valid subscription to New Relic FSO Enterprise or New Relic-approved subscription;
-  
-  4. **Authorized New Relic Endpoints**: Customer must send its data only to [New Relic’s FedRAMP-designated endpoints](/docs/security/security-privacy/compliance/fedramp-compliant-endpoints); and
-  
-  5. **Authorized Services and Features**: Customer must only use FedRAMP audited and authorized New Relic Services and features as specified [here](/docs/security/security-privacy/compliance/regulatory-audits-new-relic-services).
-</Callout>
+## Customer FedRAMP obligations [#obligations]
+
+New Relic customers must meet all of the following requirements for New Relic’s FedRAMP environment:
+
+  1. **New Relic-approved customers**: New Relic’s FedRAMP-Moderate authorized environment is only available to New Relic-approved customers. For more information, contact your New Relic account representative.
+
+  2. **Order form**: Customer’s order form with New Relic must include customer’s eligibility for FedRAMP.
+
+  3. **Subscription level**: Customer must have a current and valid subscription to New Relic Full-Stack Observability Enterprise or New Relic-approved subscription.
+
+  4. **Authorized New Relic endpoints**: Customer must send its data only to [New Relic’s FedRAMP-designated endpoints](/docs/security/security-privacy/compliance/fedramp-compliant-endpoints).
```

Comment:
"must only"

Reply:
Also, this link is circular and links back to this page. Not sure why we need this link?

Reply:
touche